src/kani-compiler/rustc_codegen_kani/src/overrides/hooks.rs

```diff
@@ -98,7 +98,13 @@ impl<'tcx> GotocHook<'tcx> for ExpectFail {
         let target = target.unwrap();
         let cond = fargs.remove(0).cast_to(Type::bool());
         //TODO: actually use the error message passed by the user.
+
+        // msg is the comment field that's set on source location
         let msg = "EXPECTED FAIL";
+
+        // property_class is used as a unique identifier for this assert
+        let _property_class = "expect_fail";
+
```

Comment:
property_class is unused for now, but when we use `assert_stmt` instead of `assert`, it needs to be passed as the second variable.